### PR TITLE
Create akimbo.json

### PR DESCRIPTION
**Pull request by aksh120**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] The file's name is all lowercased and alphanumeric. 
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/akimbo.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "Porfolio website for Akimbo,and Pixel Peak Studio",
+  "repo": "https://github.com/aksh120",
+  "owner": {
+    "username": "aksh120",
+    "email": "projects.akimbo@gmail.com"
+  },
+  "record": {
+    "NS": [
+      "casey.ns.cloudflare.com",
+      "rosalie.ns.cloudflare.com"
```
Comment (CuteDog5695) [inline domains/akimbo.json:11]: Please give a reason as to why you want NS records.

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/akimbo.json (hunk 1 above)
Comment (aksh120) [reply domains/akimbo.json:11]: To link it with cloudflare.

Comment (CuteDog5695) [reply domains/akimbo.json:11]: Not a valid reason.

**Review by iostpa — CHANGES_REQUESTED** [review]
Reason is invalid, do you want to replace it with another record or nah?

**Comment by CuteDog5695** [comment]
By other records I mean like anything other than NS records (since you don't have a valid reason for it) and just plain TXT (because it would be pointing to nothing almost).

File: domains/akimbo.json (hunk 1 above)
Comment (aksh120) [reply domains/akimbo.json:11]: This ns leads to cloudflare once done I will connect it to my plesk server

Comment (CuteDog5695) [reply domains/akimbo.json:11]: I don't think thats a valid reason still.

Comment (aksh120) [reply domains/akimbo.json:11]: So what do I do? To connect it with cloudflare

Comment (CuteDog5695) [reply domains/akimbo.json:11]: You need a valid reason for it, and as far as I've seen, you don't have one.

Comment (CuteDog5695) [reply domains/akimbo.json:11]: Reason why I need a valid reason is so that it doesn't get used for malicious stuff.

Comment (aksh120) [reply domains/akimbo.json:11]: What reason can I give I just want to make a portfolio website of mine

**Pull request closed without merging**